### ISSUE TITLE
View settings while VM is running

### ISSUE THREAD

**Issue by sampscl** [utmapp/UTM#4220]
**Describe the issue**
The settings button in disabled when the VM is running. It would be really handy to be able to just view settings without having to stop the VM first.

**Configuration**
* UTM Version: 3.2.4
* OS Version: 13.0 (Ventura beta)
* Intel or Apple Silicon? M1

**Comment by osy** [comment]
This was considered in #133 but most of the settings must take a reboot to apply anyways so it wasn't thought to be needed. With the new settings backend it would be a lot easier to do this but I'm not sure about the utility. Open to discussion about it though.

**Comment by sampscl** [comment]
> 

To me, this is a convenience item. I've got a bunch of VMs that have multiple network adapters attached to them and verifying which VM has which subset of the available adapters assigned and pairing those up with the Mac's list of adapters is tedious and error prone; just being able to see the command line for qemu would be enough!

**Comment by GiorgioComitini** [comment]
>  It would be really handy to be able to just view settings without having to stop the VM first.


Agreed, I would like this feature too. Not the live updates of settings, just to be able to view them.

**Comment by sbates130272** [comment]
+1. This is very useful and seems comparable to the XML dumping modes that virsh provides.

**Comment by freebrowser1** [comment]
This is also annoying when I cannot even view (yes, view, read-only !) the VM settings while that VM is running.

**Comment by fwarmuth** [comment]
Yes, viewing is super handy.
Also changing networks (e.g detaching) should ne request a reboot.

**Comment by disco-funk** [comment]
It's hard to imagine that anyone thinks that this feature is "not needed" or they are "not sure about the utility". Perhaps those opinions were left by someone who isn't a daily user of multiple VMs or who creates and trashes 20, 30 VMs a day when testing something. There are a multitude of reasons why the user might want to quickly click on a button, page or tab somewhere to reveal what settings the running VM has. Yes we can use the command line for QEMU, or go into the guest OS settings (who generally have this info split up into several different places). But most users would prefer the convenience of pressing a single button to see all the settings in one place. 

"This VM is running dog slow.. Let me just check quickly how big the hard drive that I provisioned was, how much RAM i gave it and how many cores the CPU has... and now quickly do a visual comparison to that other VM that's running fine."
That's the utility. It's a basic.
